tools/model.go

```diff
@@ -337,8 +342,8 @@ func NewRasModel(key string, fs filestore.FileStore) (*RasModel, error) {
 			rasWG.Flow.Add(1)
 			go getFlowData(&rm, fp, &rasWG.Flow)
 
-		case rasRE.Projection.MatchString(ext):
-			if filepath.Base(key) != filepath.Base(fp) {
+		case rm.Metadata.Projection == "" && rasRE.Projection.MatchString(ext):
```

Comment:
What if there are two projection files, whether those are .prj or .projection in the same directory as the model .prj file, you don't want the user to know? I didn't think this actually caused an error, just printed the error.